nucypher/network/nodes.py

```diff
@@ -398,9 +404,18 @@ def remember_node(self, node, force_verification_check=False, record_fleet_state
                 # This node is already known.  We can safely return.
                 return False
 
+        try:
+            stranger_certificate = node.certificate
+        except AttributeError:
+            # Whoops, we got an Alice, Bob, or someone...
+            raise self.NotATeacher(f"{node.__class__.__name__} cannot be remembered.")
```

Comment:
I think, we should say that the node didn't have a certificate explicitly in the logs

Reply:
Yeah, Good Idea.

Reply:
:heavy_check_mark: